src/generated/lcf/lsd/chunks.h

```diff
@@ -737,7 +737,13 @@ namespace LSD_Reader {
 			/**  */
 			keyinput_timed = 0x29,
 			/** Used for a wait command WaitForKeyInput rm2k3 feature to wait for decision key press. */
-			wait_key_enter = 0x2A
+			wait_key_enter = 0x2A,
+			/**  */
```

Comment:
Comments from the .csv are missing here. Those were probably applied afterwards without re-generating the files?